### PR TITLE
prettier: add --write example for js and ts files

### PR DESCRIPTION
**Pull request by AMATEURTOSS**
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

Added an example to format only for specific file extensions. I think this might be useful

### CODE REVIEW

File: pages/common/prettier.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Format files or directories recursively using single quotes and no trailing commas:
 
 `prettier --single-quote --trailing-comma {{none}} --write {{path/to/file_or_directory}}`
+
+- Format files with matching extensions:
```
Comment (CleanMachine1) [inline pages/common/prettier.md:26]: Is this in the current directory.

Comment (AMATEURTOSS) [reply pages/common/prettier.md:26]: `**/*.{js,jsx,ts,tsx}` contain
* `foo.js`
* `bar.ts`
* `foo/bar.js`
* `foo/bar/baz.tsx`

Match whatever is in the current directory

Comment (navarroaxel) [reply pages/common/prettier.md:26]: I think this is an specific usage of the 4th example
```suggestion
- Format JavaScript and TypeScript files recursively, replacing the original:
```

File: pages/common/prettier.md [hunk 2]
```diff
@@ -22,3 +22,7 @@
 - Format files or directories recursively using single quotes and no trailing commas:
 
 `prettier --single-quote --trailing-comma {{none}} --write {{path/to/file_or_directory}}`
+
+- Format JavaScript and TypeScript files recursively, replacing the original:
+
+`prettier --write "**/*.{js,jsx,ts,tsx}"`
```
Comment (CleanMachine1) [inline pages/common/prettier.md:28]:
```suggestion
`prettier --write "**/*.{{js,jsx,ts,tsx}}"`
```

**Review by CleanMachine1 — COMMENTED** [review]
Not sure if this is correct.

File: pages/common/prettier.md (hunk 2 above)
Comment (AMATEURTOSS) [reply pages/common/prettier.md:28]: `prettier --write "**/*.{js,jsx,ts,tsx}"` is whole command. 
if you highlight it, you can highlight `prettier --write "**/*.{{{js,jsx,ts,tsx}}}"` like this

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**